packages/react-devtools-shared/src/__tests__/utils-test.js

```diff
@@ -37,4 +45,44 @@ describe('utils', () => {
       expect(getDisplayName(FauxComponent, 'Fallback')).toEqual('Fallback');
     });
   });
+  describe('getDisplayNameForReactElement', () => {
+    it('should return correct display name for an element with function type', () => {
+      function FauxComponent() {}
+      FauxComponent.displayName = 'OverrideDisplayName';
+      const FauxElement = {type: FauxComponent};
```

Comment:
Why not import React and test actual elements here and below?

Reply:
Ok!